CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # ChangeLog
 
+## Unreleased
+
+- decoupled session store from `auth-provider` feature
+  `ft_sdk::session` module can be used to interact with the session store
+- `ft_sdk::utils::uuid_v8` function to generate uuids.
+- impl `Display` for `ft_sdk::PlainText`.
+- derive `Clone` for `ft_sdk::Cookie` and `ft_sdk::Host`
```

Comment:
Document that we are moving `ft_sdk::auth::SessionID` to `ft_sdk::session::SessionID`. Should we move it to `ft_sdk::SessionID` instead?

Reply:
done in https://github.com/FifthTry/ft-sdk/pull/52/commits/30948ffd97d2024e4344769673b528685e65dda5